### ISSUE TITLE
This is awesome!

### ISSUE THREAD

**Issue by WDIlsb** [macroxue/2048-ai#1]
To tell the truth, I don't fully understand all those clever words in description, but I used this repository many times to get first place in different places.

This solution probably the best one among other github repos. 

So I just wanted to let you know that there is at least one person who very thankful for your work, best wishes!

**Comment by macroxue** [comment]
Cool! It's great that you are having fun with the AI. Thanks for letting me know.

**Comment by WDIlsb** [comment]
Very glad about the latest update with "big runs".

I even managed to run it on my Raspberry Pi 4 (8 ram). The hardest part was about generating "10a" and "11b" tuple moves, but luckily I have my main PC 32 ram, so I used it and then transferred these tuples to raspberry via SFTP.

Works great, it even consumes less ram than the previous one, this way I can run 2 instances at once (or maybe even more).

Thank you again, @macroxue !
![image](https://user-images.githubusercontent.com/59749304/167781941-49938393-10d8-4f4c-92ee-df99cea60340.png)

**Comment by macroxue** [comment]
Yeah, the new code only loads lookup data on demand so the resident memory is much less than the virtual memory, as shown in your picture. I've been running as many instances as the number of CPUs on my computer. 

I see you're running with depth 9, which is super slow and may have diminished return. Depth 6 is a sweet spot balancing quality and speed. 

Have fun!

**Comment by Googleaccountreset** [comment]
How to fork github in mobile

**Comment by elexunix** [comment]
Hello @macroxue I am trying to reach you using macroxue@yahoo.com as written in your bio, but gmail says this email doesn't exist.. How can I reach you?
Your project is awesome!

**Comment by macroxue** [comment]
I haven't been using the yahoo email for some time. Is there anything I can do for you? Maybe open a new ticket?

**Comment by elexunix** [comment]
I have some computational resources (96-core server with 570GB RAM), and you mentioned in your readme that they would help to advance your project further and make an even stronger program
I can share them with you
Could you please give me your email or write me (elexunix@gmail.com)?

**Comment by macroxue** [comment]
Alex, I contacted you from macroxue@hotmail.com on Tuesday. Not sure if you received the message.

**Comment by elexunix** [comment]
Oh, now I see it!
I am very sorry for missing your email at first -- it turned out gmail automatically moved it to the spam folder...
